Modules/Filtering/CurvatureFlow/itk-module.cmake

```diff
@@ -2,7 +2,8 @@ set(DOCUMENTATION "This module contains filters that implement variations of
 Curvature Flow. This is a technique that uses an iterative solution of partial
 differential equations to implement image denoising image filtering. These
 classes are typically used as edge-preserving smoothing filters. You may also
-find the ITKSmoothing and the ITKAnisotropicSmoothing useful as well.")
+find the \\\\ref ITKSmoothing and the \\\\ref ITKAnisotropicSmoothing useful
```

Comment:
Perhaps add the word "module" here, as in `the \\\\ref ITKSmoothing and the \\\\ref ITKAnisotropicSmoothing module` ?

Reply:
ITK's scripts automatically add a `Module` prefix to these, so no need for that. Please have a look at the Doxygen files.